libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/CohortStatsHeatmap.tsx

```diff
@@ -132,11 +113,11 @@ export class CohortStatsHeatmap extends React.Component<
               columns: [
                 {
                   labels: {
-                    // format labels to cap the line length
                     formatter() {
-                      return wrapYAxisLabels(this.value.toString(), true);
+                      const text = wrapYAxisLabels(this.value.toString());
+                      return `<div style='width:300px'>${text}</div>`;

```

Comment:
Does this formatter only take html string? not react element?

Reply:
Correct. You have two choices:
- a function (which is what I used) that returns a string
- just a format string (so not even a function) in a slightly differently named parameter